Prefixed table names with replicache_

nbarraud opened the pull request:
I prefixed the `space`, `entry`, `client` and `meta` tables with `replicache_` to avoid conflicts with existing userland tables.

I haven't touched the version number but I think the major should be bumped by one since that's a breaking change.

Commit:
Prefixed table names with replicache_

nbarraud commented:
I haven't touched the version number but I would think the major should be bumped one since that's a breaking change.

Commit:
Fixed error in create table script

Commit:
Prefixed meta table

Commit:
Prefixed meta table

cesara reviewed (changes requested):
Great work! Please fix failing test. In data.test.ts it should have the following expected output for the `createSpace` test. 
Please rebase with latest from rocicorp main aswell. Thanks again!

```
        expect(String(e)).contains(
          `duplicate key value violates unique constraint "replicache_space_pkey`
        );
```

Commit:
Fixed test error

nbarraud commented:
Rebased, made the test fix and also added the rimraf package for cross-OS compatibility (rm -rf not available on Windows). Bumped the patch version as well.

Thank you!
